src/api/lib/tasks/dev.rake

```diff
@@ -263,6 +263,9 @@ namespace :dev do
       # Create a request with a delete request action
       Rake::Task['dev:requests:request_with_delete_action'].invoke
 
+      # Create a request with a delete release action
```

Comment:
This comment should be modified... It looks like a copy-paste from the one above.